### DISCUSSION TITLE
Using Redis in windows 10 without docker.

### DISCUSSION THREAD

**Discussion by manupatel007** [django/channels#1623] [Q&A] [answered]
I followed the django-channels tutorial and in place of docker, I tried to use redis directly on my Windows 10 machine. I found that there is no official support for windows. So I followed a workaround on stackoverflow which pointed to download the latest redis version for windows from [here](https://github.com/tporadowski/redis/releases) and it worked. So I wanted to ask if it is a valid way to work with django-channels.(Pardon me if I asked this at wrong place).

**Comment by carltongibson** [comment, marked as answer]
If you can get redis running then it should work. 

It might be easy to use the WSL? I'm afraid I really can't speak to the compatibility of Redis with Windows.

**Reply by manupatel007** [reply]
Thanks for this. Actually I tried both ways(WSL and one specified above). But the WSL seemed to be much resource heavy since only redis was needed to run.
Also, if sole reason for using docker in tutorial is to run redis irrespective of the operating system, Is it a good idea to add a little "why?" section telling why we use docker, not directly use redis.
